sdk/storage/azfile/sas/service.go

```diff
@@ -38,8 +39,8 @@ type FileSignatureValues struct {
 
 // SignWithSharedKey uses an account's SharedKeyCredential to sign this signature values to produce the proper SAS query parameters.
 func (v FileSignatureValues) SignWithSharedKey(sharedKeyCredential *SharedKeyCredential) (QueryParameters, error) {
```

Comment:
It seems a bit odd to call this `FileSignatureValues` when it supports files and shares.  What about simply `SignatureValues`?

Reply:
Its like how we have in azblob where `BlobSignatureValues` supports both container and blobs.
I can rename it to `SignatureValues` if that's what we want in azfile.

Reply:
Done